frontend/src/pages/destructionlist/detail/pages/process-review/DestructionListProcessReviewPage.tsx

```diff
@@ -137,6 +146,93 @@ export function DestructionListProcessReviewPage() {
     }));
   }, [reviewItems, zaakReviewStatuses]);
 
+  /**
+   * Get the process review badge for a zaak to know what mutation is proposed.
+   */
+  function getProcessReviewBadge(zaak: Zaak) {
+    const processZaakReviewDetail =
+      processZaakReviewSelectionState?.[zaak.url as string]?.detail;
+    const action = processZaakReviewDetail?.action;
+
+    if (!action) {
+      return (
+        // @ts-expect-error - style props not supported (yet?)
+        <Badge style={{ display: "block" }}>
+          <Solid.QuestionMarkCircleIcon />
+          Geen Mutatie
+        </Badge>
+      );
+    }
+
+    const { archiefactiedatum, selectielijstklasse, comment } =
+      processZaakReviewDetail;
+
+    const getLabel = (value: string | undefined) =>
+      selectieLijstKlasseChoicesMap?.[zaak.url]?.find(
+        (choice) => choice.value === value,
+      )?.label;
+
+    const tooltipData = {
+      archiefactiedatum,
+      oud_archiefactiedatum: zaak?.archiefactiedatum,
+      selectielijstklasse: getLabel(selectielijstklasse),
+      oud_selectielijstklasse: getLabel(zaak.selectielijstklasse),
+      opmerking: comment,
+    };
+
+    // Filtering logic:
+    // 1. Keep 'oud_' values only if the new value exists and is different.
```

Comment:
If we use dynamic keys, can't we use something that we can process with string2Title so we can offload that to admin-ui?

Also: if we use the label "oud", can we also use the label nieuw, maybe between braces, so: "Selectielijstklasse (nieuw)"